packages/x-components/src/x-modules/queries-preview/x-module.ts

```diff
@@ -0,0 +1,25 @@
+import { XModule } from '../x-modules.types';
+import { queriesPreviewEmitters } from './store/emitters';
+import { queriesPreviewXStoreModule } from './store/module';
+import { QueriesPreviewXStoreModule } from './store/types';
+import { queriesPreviewWiring } from './wiring';
+
+/**
+ * QueriesPreview {@link XModule} alias.
+ *
+ * @public
+ */
+export type QueriesPreviewXModule = XModule<QueriesPreviewXStoreModule>;
+
+/**
+ * QueriesPreview {@link XModule} implementation. This module is auto-registered as soon as you
+ * import any component from the `queries-preview` entry point.
```

Comment:
I think here should be `queriesPreview` instead of `queries-preview`. Not 100% if it matters, though.

Reply:
I would say it should be `queries-previews` as it is the folder name where the module is defined and the path used when importing the module.

Reply:
yep, agree with @mnavarroespinosa. Actually in other x-modules we do the same. See [search-box x-module](https://github.com/empathyco/x/blob/main/packages/x-components/src/x-modules/search-box/x-module.ts#L17).